### ISSUE TITLE
Alexa throwing error every time, even upon success

### ISSUE THREAD

**Issue by elmedico27** [rgraciano/echo-sonos#96]
Every time I invoke "Alexa, ask sonos to play [song/artist/etc]" for my local library I receive a vocal response of "there was a problem with the requested skills response."  I get this response even as the requested music begins playing.  Checking the logs, it appears I was getting the error because my tasks were timing out when set to both 3 and 6 seconds, so I upped the Lambda timeout to 30 seconds unsuccessfully -- requests are now finishing at ~20 sec but still throwing the voice error and writing `Process exited before completing request` to the log.  

When they don't time out, they also throw this in the log:
`TypeError: Cannot read property '0' of undefined
at Response.<anonymous> (/var/task/index.js:714:39)
at Request.<anonymous> (/var/runtime/node_modules/aws-sdk/lib/request.js:355:18)
at Request.callListeners (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:105:20)
at Request.emit (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:77:10)
at Request.emit (/var/runtime/node_modules/aws-sdk/lib/request.js:668:14)
at Request.transition (/var/runtime/node_modules/aws-sdk/lib/request.js:22:10)
at AcceptorStateMachine.runTo (/var/runtime/node_modules/aws-sdk/lib/state_machine.js:14:12)
at /var/runtime/node_modules/aws-sdk/lib/state_machine.js:26:10
at Request.<anonymous> (/var/runtime/node_modules/aws-sdk/lib/request.js:38:9)
at Request.<anonymous> (/var/runtime/node_modules/aws-sdk/lib/request.js:670:12)`

Any help/thoughts why Alexa keeps saying there was a problem as the commands work, and how to get the error to cease?

**Comment by rgraciano** [comment]
@elmedico27 - did you get this to work?

**Comment by elmedico27** [comment]
Honestly, I haven't tried recently.  I uninstalled everything, as I wasn't totally convinced it wasn't something with my setup.  I was also having problems with jishi's app generating the music library, so I took some time to troubleshoot that and clean up my library from odd characters in filenames, which I think I got all squared away.

I'll run back through the setup/install this week and let you know if I encounter the error again.
